docs/docs-content/automation/palette-sdk/palette-sdk.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: "Palette Go SDK"
+title: "Palette Go SDK"
+description: "Learn how to get started with the Palette Go SDK."
+hide_table_of_contents: false
+tags: ["palette-sdk"]
+---
+
+The Palette Go Software Developer Toolkit (SDK) allows Go developers to interact with several public Palette APIs
+through user-friendly client configurations.
+
+The SDK comprises two repositories:
+
+- The [Palette API Go](https://github.com/spectrocloud/palette-api-go) repository contains API client models that are
+  generated from Palette's public APIs.
+- The [Palette SDK](https://github.com/spectrocloud/palette-sdk-go) repository leverages the Palette API Go repository
+  to provide client configuration for interacting with Palette public APIs. Refer to the
+  [client.go](https://github.com/spectrocloud/palette-sdk-go/blob/main/client/client.go) file for a complete list of
+  client configuration options.
+
+## Get Started
+
+Check out the [List Clusters with Palette Go SDK](./list-clusters-sdk.md) example to learn how to install, configure,
+and use the SDK. This guide provides instructions and sample code for listing the active clusters in your Palette
```

Comment:
Is there anywhere we can see all the Palette operations that our SDK supports? Would be good to provide an extra resource where users can see what they can do.

Reply:
From what I understood, the client folder has all the operations. I will link to the folder :)